### PR TITLE
Fix absolute to relative path

### PR DESCRIPTION
**Pull request by MarkusTiede**
Thank you for opening a pull request and contributing to asciidoctor-maven-plugin!

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Refactor
- [ ] Build improvement
- [ ] Other (please describe)

**What is the goal of this pull request?**

**Are there any alternative ways to implement this?**

**Are there any implications of this pull request? Anything a user must know?**

**Is it related to an existing issue?**
 
- [ ] Yes
- [x] No

*Finally, please add a corresponding entry to CHANGELOG.adoc*

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -86,7 +86,7 @@ endif::[]
 
 These are the minimal steps to convert your AsciiDoc documents to HTML with the asciidoctor-maven-plugin.
 
-. Place your Asciidoc sources in [.path]_/src/docs/asciidoc_.
+. Place your Asciidoc sources in [.path]_src/docs/asciidoc_.
```
Comment (mojavelinux) [inline README.adoc:89]: Good catch! While you are on this line, could you also change Asciidoc to AsciiDoc?

Comment (MarkusTiede) [reply README.adoc]: Sure - https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/592/commits/96af436ebe20f017f9d3b0ee6361d00c964c4ed1

**Review by mojavelinux — APPROVED** [review]

**Comment by abelsromero** [comment]
I feel sorry for asking, but wouldn't be nice to take credit since you spoted it. The same section is replicated in `usage.adoc`, is it too much to ask to fix it too?

**Comment by MarkusTiede** [comment]
Sure - @abelsromero : thanks for noticing!

**Comment by abelsromero** [comment]
Thanks!

**Pull request merged**